launch/launch/actions/register_event_handler.py

```diff
@@ -32,10 +32,11 @@ class RegisterEventHandler(Action):
     place.
     """
 
-    def __init__(self, event_handler: EventHandler, **kwargs) -> None:
+    def __init__(self, event_handler: EventHandler, handle_once: bool = False, **kwargs) -> None:
         """Constructor."""
```

Comment:
be good to add docs for `handle_once` here also

Reply:
👍